schema/3pse/file/applications/espresso/7.2/pw.x/atomic_positions.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$id": "3pse/file/applications/espresso/7.2/pw.x/atomic-positions",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "atomic positions schema",
+    "description": "https://www.quantum-espresso.org/Doc/INPUT_PW.html#idm1493",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "card": {
+            "type": "string",
+            "enum": [
+                "alat",
+                "bohr",
+                "angstrom",
+                "crystal",
+                "crystal_sg"
+            ],
+            "default": "alat"
+        },
+        "values": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                    "X": {
+                        "type": "string",
+                        "description": "label of the atom as specified in ATOMIC_SPECIE"
+                    },
+                    "x": {
+                        "type": "number",
+                        "description": "atomic positions"
+                    },
+                    "y": {
+                        "type": "number",
+                        "description": "atomic positions"
+                    },
+                    "z": {
+                        "type": "number",
+                        "description": "atomic positions"
+                    },
+                    "if_pos(1)": {
+                        "type": "number",
+                        "default": 1,
+                        "minimum": 0,
+                        "maximum": 1,
```

Comment:
1. In general, there's a lot of duplication in how we define each key, we should identify reusable schema bits and isolate them, then use here. This one could be something like `integer_one_or_zero`
2. We should restrict the values here to only 0 or 1, so put "integer" as a type